internal/service/alarms/api/openapi.yaml

```diff
@@ -576,6 +677,23 @@ components:
         perceivedSeverity:
             $ref: '#/components/schemas/PerceivedSeverity'
 
+    AlarmServiceConfiguration:
+      type: object
+      properties:
+        retentionPeriod:
+          type: integer
+          description: | 
+            Number of days for alarm history to be retained.
+            This value has a minimum limit such that it can’t be set to a value lower than the limit.
```

Comment:
We should state what the minimum is.

Reply:
yeah, I know, but I didn´t want to have to maintain the constant in code and here. But yeah, will add it